JASP-Desktop/components/JASP/Widgets/CustomContrastsTableView.qml

```diff
@@ -25,21 +25,25 @@ BasicThreeButtonTableView
 {
 	id				: customContrastsTV
 
-	itemType		: "double"
-	modelType		: "CustomContrasts"
-	cornerText		: ""
-	minimum			: -Infinity
-	decimals		: 3
+	preferredHeight	: Math.max(150 * preferencesModel.uiScale,10 * preferencesModel.uiScale + tableView.y + tableView.tableHeight)
 
+	tableType			: "userDataInput"
+	itemType			: "string"
+	modelType			: "CustomContrasts"
+	initialColumnCount	: 1
+	initialRowCount		: 0
+	buttonsInRow		: true
+
+	cornerText			: ""
 
 	buttonAddText		: qsTr("Add Contrast")
-	onAddClicked		: tableView.addRow()
+	onAddClicked		: (modelType == "AnovaCustomContrasts") ? tableView.addRow() : tableView.addColumn()
```

Comment:
Maybe it would be nicer to be able to set `tableView` in column or rowmode and it just has `removeElement`, `elementCount`, `addElement` functions. 
Although I wouldn't necessarily suggest to do that here and now in this PR.

Reply:
Yes, I will make an issue to improve TableView (and BoundQMLTableView) code.
I have not time to do that now (Frantisek needs this PR now).

Reply:
Yeah no hurry